components/event-sources/Makefile

```diff
@@ -159,17 +164,8 @@ injection: bin/injection-gen
 		--versioned-clientset-package $(PKG)/client/generated/clientset/internalclientset \
 		--external-versions-informers-package $(PKG)/client/generated/informer/externalversions
 
-istio-injection: bin/injection-gen
```

Comment:
As `istio-injection` is removed, which target does its job?

Reply:
Reverted

Reply:
Please revert the whole target definition.

Reply:
I didn't noticed, ups :)